root/tree/dataframe/RDataFrameBenchmarks.cxx

```diff
@@ -135,49 +135,49 @@ static void BM_RDF_FillHistoRangeDeductionNoRDF(benchmark::State &state)
          h.Fill(i);
    }
 }
-BENCHMARK(BM_RDF_FillHistoRangeDeductionNoRDF)->Unit(benchmark::kMicrosecond)->Arg(100)->Arg(100000);
+BENCHMARK(BM_RDataFrame_FillHistoRangeDeductionNoRDF)->Unit(benchmark::kMicrosecond)->Arg(100)->Arg(100000);
 
 // Fill a histogram from a single thread
 // Data is generated on the fly, not read from a file
 // The overhead of generating the data and passing it to the Histo action should be
 // the one measured by BM_RDF_NoOpLoopOnCustomColumn
```

Comment:
Could you update this comment to the renamed bm.

Reply:
Done!